package.json

```diff
@@ -54,14 +57,18 @@
   "license": "MIT",
   "description": "Manage docker-compose from Node.js",
   "devDependencies": {
+    "@types/jest": "^24.0.15",
+    "@types/node": "^12.6.2",
+    "@typescript-eslint/eslint-plugin": "^1.12.0",
+    "@typescript-eslint/parser": "^1.12.0",
     "composefile": "^0.3.0",
     "dockerode": "^2.5.7",
     "eslint": "^5.6.0",
     "eslint-config-es": "^0.8.11",
     "eslint-plugin-extended": "^0.2.0",
     "eslint-plugin-jest": "^22.6.4",
     "jest": "^24.8.0",
-    "tape": "^4.6.3",
```

Comment:
looks like `jest-cli` is required here

Reply:
As said in general discussion comment, it's only needed if you want to call `./node_modules/.bin/jest`. However it's probably better to use `npx jest` or `npm run test` (since it's defined in `package.json`) to invoke tests.

Reply:
`npm test` after `npm install` returns this (using Node 10.15.1 via `nvm`):

```bash
npm test

> docker-compose@0.19.0 test /Users/alexzeitler/src/pdmlab-oss/docker-compose-83
> jest test --verbose

sh: jest: command not found
npm ERR! Test failed.  See above for more details.
```